cms/djangoapps/contentstore/tests/test_course_listing.py

```diff
@@ -107,6 +109,40 @@ def test_errored_course_global_staff(self):
             courses_list_by_groups, __ = _accessible_courses_list_from_groups(self.request)
             self.assertEqual(courses_list_by_groups, [])
 
+    @ddt.data(
+        (ModuleStoreEnum.Type.split, 5),
+        (ModuleStoreEnum.Type.mongo, 3)
```

Comment:
A useful technique is to add performance tests like this as a separate commit first, to characterize how bad the old way is, and then have another commit with the performance improvements.